packages/devtools/lib/src/main.dart

```diff
@@ -197,10 +214,16 @@ class PerfToolFramework extends Framework {
       }
     });
 
-    final ActionButton restartAction =
-        ActionButton('icons/hot-restart-white@2x.png', 'Hot Restart');
+    addGlobalAction(reloadAction);
+  }
+
+  void _buildRestartButton() async {
+    final ActionButton restartAction = ActionButton(
+      'icons/hot-restart-white@2x.png',
```

Comment:
nit: you should define these as ImageIcon classes. See icons.dart
That way the 2x version can be used on high DPI displays and the 1x version can be used elsewhere. Create a 1x version (without the @2x) in the name if there isn't one.